recon_linux.go

```diff
@@ -43,17 +44,16 @@ func (irc *IrcCon) StartUnixListener() {
 }
 
 // Attempt to hijack session previously running bot
-func (irc *IrcCon) HijackSession() bool {
+func (irc *Bot) hijackSession() bool {
 	unaddr, err := net.ResolveUnixAddr("unix", irc.unixastr)
 	if err != nil {
-		irc.Log(LWarning, "could not resolve unix socket")
+		irc.Error("could not resolve unix socket")
```

Comment:
This is probably a programmer error, since the only possible error case is when the first argument is neither of "unix", "unixgram", "unixpacket": https://golang.org/src/net/unixsock.go?s=826:883#L30